pixeltable/metadata/__init__.py

```diff
@@ -24,9 +24,10 @@ def create_system_info(engine: sql.engine.Engine) -> None:
     system_md = SystemInfoMd(schema_version=VERSION)
     record = SystemInfo(md=dataclasses.asdict(system_md))
     with orm.Session(engine, future=True) as session:
-        session.add(record)
-        session.flush()
-        session.commit()
+        if session.query(SystemInfo).count() == 0:
```

Comment:
Why is this needed?

Reply:
for idempotency in cloud version we dont have a way to figure out if db is initialized or not.
So we try to add SystemInfo and create all tables again in each pass 
    metadata.schema.base_metadata.create_all(self._sa_engine, checkfirst=True)
    metadata.create_system_info(self._sa_engine)